### PR TITLE
PPTT-3254 Require semver from tracking source version

### PR DESCRIPTION
**Pull request by adaqus**

### CODE REVIEW

**Review by prk3 — CHANGES_REQUESTED** [review]
`AnonymizedTrackingParameters` in `data_collection/api/tracker_debugger_api.yaml` does not have tracking source parameters. Please correct that.

File: data_collection/api/tracking_api.yaml [hunk 1]
```diff
@@ -850,8 +850,9 @@ paths:
         - name: ts_v
           in: query
           description: |
-            Version of a library used for tracking.
+            Version of a library used for tracking. [Semantic Versioning](https://semver.org) format required.
```
Comment (RomanSek) [inline data_collection/api/tracking_api.yaml:853]: If you want to link description of this term, then why only here? Other YAML files are also mentioning it.

**Review by RomanSek — APPROVED** [review]
[Preview](https://developers.piwik.pro/en/feature-pptt-3254-restrict-tracking-source-verstion-to-semver/data_collection/api/http_api.html) looks ok to me.

**Comment by adaqus** [comment]
> `AnonymizedTrackingParameters` in `data_collection/api/tracker_debugger_api.yaml` does not have tracking source parameters. Please correct that.

Fixed.

File: data_collection/api/tracking_api.yaml (hunk 1 above)
Comment (adaqus) [reply data_collection/api/tracking_api.yaml:853]: I've added links to each Semantic Versioning occurrence.

**Review by prk3 — APPROVED** [review]
Please inform people on docs channel that you're changing the port to 9011.

**Comment by adaqus** [comment]
> Please inform people on docs channel that you're changing the port to 9011.

@prk3 this is a mistake. I've changed back to 9009.

**Pull request merged**